crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/service/stateless/StatelessAssessmentServiceTest.java

```diff
@@ -0,0 +1,155 @@
+package uk.gov.justice.laa.crime.meansassessment.service.stateless;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.justice.laa.crime.meansassessment.service.AssessmentCriteriaChildWeightingService;
+import uk.gov.justice.laa.crime.meansassessment.service.AssessmentCriteriaService;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.CaseType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.Frequency;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.MagCourtOutcome;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.AgeRange;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.IncomeType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.OutgoingType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.StatelessOutcome;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.repository.AssessmentCriteriaChildWeightingRepository;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.repository.AssessmentCriteriaDetailFrequencyRepository;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.repository.AssessmentCriteriaRepository;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.repository.CaseTypeAssessmentCriteriaDetailValueRepository;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+@DataJpaTest
```

Comment:
I don't think there's any need to load the application context here, we can simply mock the dependencies. See the existing assessment service for an example.

Reply:
I didn't want to do that. I wanted to prove that I'm calling all the right code correctly.

Reply:
This is now an integration test as suggested

Reply:
I'm confused - I've looked and the name has got integration tests in it (IntelliJ did the rename IIRC)

Reply:
You're right, I can see it when checking out the branch, but not on GitHub, weird!